### DISCUSSION TITLE
Checkbox field type for form

### DISCUSSION THREAD

**Discussion by frankomator** [Codeinwp/otter-blocks#1509] [Ideas & Feedback]
Feature request: It would be great to have a checkbox field type in the form block, as it is often necessary to ask for consent to collect and save personal data (GDPR compliance). And if the form could save the entries to the database/WordPress backend, it could be a replacement for other form plugins.

Best,
Frank

**Comment by HardeepAsrani** [comment]
@frankomator Both of these features are already in works and it will be available soon in the Form block.
